course/models.py

```diff
@@ -450,6 +450,11 @@ def __str__(self):
     def clean(self):
         super().clean()
         errors = {}
+        RESERVED = ("instances")
```

Comment:
This is not a tuple. One-element tuple definition requires the trailing comma `,`. The `in` operator behaves differently for strings and tuples, since strings match substrings while tuples require full equality with one element.

```python
>>> a = "daad"
>>> type(a)
<class 'str'>
>>> b = ("dadas")
>>> type(b)
<class 'str'>
>>> c = ("adsad",)
>>> type(c)
<class 'tuple'>
>>> "da" in b
True
>>> "dadas" in b
True
>>> "ad" in c
False
>>> "adsad" in c
True
```

Reply:
Thanks, great feedback